vector-im/element-android
# Voice Broadcast - Listening

## Type of change


## Content

Add the ability to listen to a voice broadcast.
For the moment, the incoming voice messages are not added to the queue when the listening is ongoing.

## Motivation and context

Continue #7127 

## Screenshots / GIFs


## Tests

- Enable the voice broadcast feature flag
- Start a voice broadcast in a room
- Verify that you can listen to a voice broadcast from another account when it has ended
- Verify that you can listen to an ongoing voice broadcast from another account (only for the already received events)
- Verify that the UI is correctly updated

## Tested devices

- OS version(s):

## Checklist
